app/geckoNightly/fennecNightly/output.json

```diff
@@ -0,0 +1 @@
+[{"outputType":{"type":"APK"},"apkData":{"type":"FULL_SPLIT","splits":[{"filterType":"ABI","value":"arm64-v8a"}],"versionCode":2015711683,"versionName":"","enabled":true,"filterName":"arm64-v8a","outputFile":"app-geckoNightly-arm64-v8a-fennecNightly.apk","fullName":"geckoNightlyArm64-v8aFennecNightly","baseName":"geckoNightly-arm64-v8a-fennecNightly"},"path":"app-geckoNightly-arm64-v8a-fennecNightly.apk","properties":{}},{"outputType":{"type":"APK"},"apkData":{"type":"FULL_SPLIT","splits":[{"filterType":"ABI","value":"x86"}],"versionCode":2015711685,"versionName":"","enabled":true,"filterName":"x86","outputFile":"app-geckoNightly-x86-fennecNightly.apk","fullName":"geckoNightlyX86FennecNightly","baseName":"geckoNightly-x86-fennecNightly"},"path":"app-geckoNightly-x86-fennecNightly.apk","properties":{}},{"outputType":{"type":"APK"},"apkData":{"type":"FULL_SPLIT","splits":[{"filterType":"ABI","value":"armeabi-v7a"}],"versionCode":2015711681,"versionName":"","enabled":true,"filterName":"armeabi-v7a","outputFile":"app-geckoNightly-armeabi-v7a-fennecNightly.apk","fullName":"geckoNightlyArmeabi-v7aFennecNightly","baseName":"geckoNightly-armeabi-v7a-fennecNightly"},"path":"app-geckoNightly-armeabi-v7a-fennecNightly.apk","properties":{}},{"outputType":{"type":"APK"},"apkData":{"type":"FULL_SPLIT","splits":[{"filterType":"ABI","value":"x86_64"}],"versionCode":2015711687,"versionName":"","enabled":true,"filterName":"x86_64","outputFile":"app-geckoNightly-x86_64-fennecNightly.apk","fullName":"geckoNightlyX86_64FennecNightly","baseName":"geckoNightly-x86_64-fennecNightly"},"path":"app-geckoNightly-x86_64-fennecNightly.apk","properties":{}}]
```

Comment:
Seems like this might have been included by accident?

Reply:
Yup, thanks!